broker/src/main/java/org/apache/rocketmq/broker/failover/EscapeBridge.java

```diff
@@ -63,44 +96,52 @@ public void start() throws Exception {
         if (brokerController.getBrokerConfig().isEnableSlaveActingMaster() && brokerController.getBrokerConfig().isEnableRemoteEscape()) {
             String nameserver = this.brokerController.getNameServerList();
             if (nameserver != null && !nameserver.isEmpty()) {
-                startInnerProducer(nameserver);
-                startInnerConsumer(nameserver);
-                LOG.info("start inner producer and consumer success.");
+                this.defaultAsyncSenderExecutor = new ForkJoinPool(Runtime.getRuntime().availableProcessors(),
+                        new NamedForkJoinWorkerThreadFactory("AsyncEscapeBridgeSender_", brokerController.getBrokerIdentity()),
+                        null,
+                        false);
```

Comment:
Why do we need to create a ForkJoinPool? I think to create a common thread pool, and also do not need to create a NamedForkJoinWorkerThreadFactory util class

Reply:
I've replace it to normal ThreadPoolExecutor. ForkJoinPool brings no much benefits as it is not a divide-and-conquer situation.